cmd/gke-exec-auth-plugin/tpm.go

```diff
@@ -136,6 +129,15 @@ func tpmAttest(dev tpmDevice, privateKey crypto.PrivateKey) ([]byte, error) {
 		return nil, fmt.Errorf("verifying certification signature with AIK cert: %v", err)
```

Comment:
They should be the same, but not necessarily. There are 2 EK certs - one for EK with default template (per the spec) and one for signing EK, which we actually use.
If we loaded the former EK cert, this would fail, surfacing our bug.